src/analog/constants/channelsNames.ts

```diff
@@ -0,0 +1,5 @@
+import * as channelsTypes from './channelsTypes.js';
```

Comment:
would be nice to add all these new const files to src/analog/constants/index.ts

Reply:
fixed